src/AssemblerHttpServer.cpp

```diff
@@ -839,6 +839,8 @@ void Assembler::writeAssemblySummaryBody(ostream& html)
         "<td class=right>" << assemblerInfo->assemblyElapsedTimeSeconds/3600. <<
         "<tr><td>Average CPU utilization"
         "<td class=right>" << assemblerInfo->averageCpuUtilization <<
+        "<tr><td>Peak Memory utilization (bytes)"
+        "<td class=right>" << assemblerInfo->peakMemoryUsage <<
```

Comment:
For macOS, `peakmemoryUsage` will be `0`. Write `Not determined` rather than `0`, or equivalent message to that effect. Similar comment below for `json` output.

Reply:
Sure.